### ISSUE TITLE
Scale layer after Batch Normalization

### ISSUE THREAD

**Issue by notabigfish** [flyyufelix/cnn_finetune#55]
Hi, thanks for the excellent codes! 

I'm a little confused about the "Scale" layer after "BatchNormalization" layer. Why are scale parameters not directly set in "BatchNormalization()", but written in a separate class?

 Thank you!
